### DISCUSSION TITLE
phpMailer not connecting to smtp.gmail.com

### DISCUSSION THREAD

**Discussion by abubakarsd** [community/community#81616] [Programming Help]
### Body

Please is anyone having problem in the new phpmailer version
2023-12-23 17:44:27 SMTP ERROR: Failed to connect to server: Network is unreachable (101)
SMTP Error: Could not connect to SMTP host. Failed to connect to server
Message could not be sent. Mailer Error: SMTP Error: Could not connect to SMTP host. Failed to connect to serverSMTP server error: Failed to connect to server SMTP code: 101 Additional SMTP info: Network is unreachable

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by BhathikaGimhan** [comment]
**Here's a breakdown of the error message and potential causes:**

- **"SMTP ERROR: Failed to connect to server: Network is unreachable (101)"**: This indicates a network connectivity issue preventing PHPMailer from reaching the SMTP server.

You're getting a "network unreachable" error when sending emails with PHPMailer. This means your email app can't reach the email server. Here's what to do:

1. **Double-check your settings:** Make sure your email address, password, and server details are correct.
2. **Test your internet:** Check if you have a stable internet connection.
3. **Firewall & security:** Check if any software is blocking PHPMailer's connection.
4. **Email provider issues:** See if your email provider's servers are having problems.
5. **PHPMailer code:** Look for any errors in your PHPMailer code.

If you're still stuck, search online for help with your specific email provider and PHPMailer version. Good luck!

**Comment by abubakarsd** [comment]
I tried this process but it doesn't work
